be/src/vec/exec/scan/file_scanner.h

```diff
@@ -192,6 +192,15 @@ class FileScanner : public Scanner {
             _partition_col_descs;
     std::unordered_map<std::string, VExprContextSPtr> _missing_col_descs;
 
+    // store all slot descriptors, used for initializing runtime filter partition prune context
+    std::unordered_map<std::string, const SlotDescriptor*> _all_col_name_to_slot_desc;
```

Comment:
we should explain more about `_partition_slot_descs` and `_all_col_name_to_slot_desc`